README.md

```diff
@@ -132,6 +138,23 @@ _           | `host.py`                   | (to be removed) control a remote dev
 **others**  | `launch.sh`                 | shell script to start `pvm.py` with one _click_
 _           | `build_omxplayer.sh`                 | shell script to build `omxplayer` with one _click_
 
+## How to deploy your code when you testing in multiple rpis?
+Use the [deploy_code_to_rpi.sh](https://github.com/omarcostahamido/PVM/blob/AddCDscript/deploy_code_to_rpi.sh) script.
+But there are two prerequisites:
+
+One is the need to set up passwordless ssh access.
+For the passwordless ssh access please follow [here](https://danidudas.medium.com/how-to-connect-to-raspberry-pi-via-ssh-without-password-using-ssh-keys-3abd782688a).
+
+Second is the need to set the autostart.
+
+Once you're done, run the command on your computer.
```

Comment:
would be nice to also clarify here that it will deploy the latest (maybe unstable?) version of the software on all the devices whose IP number appears on the .txt file.

Reply:
Added comment.